plugins/aws-catalog-backend/src/RDSEntityProvider.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright SeatGeek
+ * Licensed under the terms of the Apache-2.0 license. See LICENSE file in project root for terms.
+ */
+import {
+  DBInstance,
+  DescribeDBInstancesCommandInput,
+  RDSClient,
+  paginateDescribeDBInstances,
+} from '@aws-sdk/client-rds';
+import { AwsCredentialIdentity } from '@aws-sdk/types';
+import { TaskRunner } from '@backstage/backend-tasks';
+import { Config } from '@backstage/config';
+import { Logger } from 'winston';
+import {
+  BaseEntityProvider,
+  InstanceFilter,
+  InstanceTransformer,
+} from './BaseEntityProvider';
+
+/**
+ * A db instance from the AWS SDK.
+ * @public
+ */
+export type RDSDBInstance = DBInstance;
+/**
+ * Input for the RDSDescribeDBInstancesCommand.
+ * @public
+ */
+export type RDSDescribeDBInstancesCommandInput =
+  DescribeDBInstancesCommandInput;
```

Comment:
fwiw, i did this here and in the okta provider as it makes it a bit easier than us rewriting the interface or having users also add the aws library when they use the provider, but it does mean we risk the following:

1. aws sdk pushes a breaking change to one of these aliased types
2. we bump the version and release as a non major change (not realizing some interface has updated)
3. user code breaks

Reply:
That's a good point.  I'm of the opinion that, because AWS also follows semver, this risk is acceptable as long as we never bump the AWS major version without also bumping our own.